objects.proto

```diff
@@ -53,17 +53,37 @@ message Federation {
   Disjunction disjunction = 1;
 }
 
-message Location {
+
+// Should only be used when quotient adds new locations to the tree
+enum SpecialLocation {
+  UNIVERSAL = 0;
+  ERROR = 1;
+}
+
+// A specific location in a specific component
+message LeafLocation {
   string id = 1;
   SpecificComponent specific_component = 2;
 }
 
-message LocationTuple {
-  repeated Location locations = 1;
+// A combination/pair of locations
+message BranchLocation {
```

Comment:
I am not a big fan of `BranchLocation`. It does represent a branching out, but it is not a location itself. Maybe `LocationFork`, which signals that the location tree is forked at the given node?

Reply:
While I agree on a name change, personally, I'd prefer `InnerLocation` since they represent inner nodes, and would conform more with `LeafLocation` :smile:.